schemas/com.iterable/system_webhook/jsonschema/1-0-0

```diff
@@ -0,0 +1,695 @@
+    "email": {
+      "type": "string",
+      "description": "The email address associated with the event",
+      "maxLength": 320,
+      "format": "email"
+    },
+    "eventName": {
+      "type": "string",
+      "description": "The name of the event for which a webhook has been triggered",
+      "maxLength": 4096
+    },
+    "dataFields": {
+      "type": "object",
+      "properties": {
+        "appAlreadyRunning": {
+          "type": [
+            "boolean",
+            "null"
+          ],
+          "description": "Whether or not the app was already running when the push notification arrived."
+        },
+        "badge": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Badge to set for push notification",
+          "maxLength": 4096
+        },
+        "bounceMessage": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "E-mail bounce message.",
+          "maxLength": 4096
+        },
+        "browserToken": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Token provided by Firebase Messaging javascript API.",
+          "maxLength": 4096
+        },
+        "campaignId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Reference to the campaign from which the event originated (e.g., 74768)"
+        },
+        "campaignName": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Campaign name",
+          "maxLength": 4096
+        },
+        "canonicalUrlId": {
+          "type": [
+            "string",
+            "null"
+          ],
+          "description": "Reference to the URL associated with the event (e.g., '3145668988')",
+          "maxLength": 4096
+        },
+        "catalogCollectionCount": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 2147483647,
+          "description": "The number of times a catalog collection was referenced in the message."
+        },
+        "catalogLookupCount": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 2147483647,
+          "description": "The number of times a catalog was referenced in the message."
+        },
+        "channelId": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "minimum": 0,
+          "maximum": 9223372036854775807,
+          "description": "Unique identifier of the channel (e.g., 2203)"
+        },
+        "channelIds": {
+          "type": [
+            "array",
+            "null"
+          ],
+          "items": {
+            "type": [
+              "integer",
+              "null"
```

Comment:
Is this right, that the array can contain nulls?  It means this is valid data:
```
[ 5, 6, null, 7]
```

Reply:
Good point, changed it not to allow nulls inside the list.